core/examples/luigi-example-openui5/README.md

```diff
@@ -0,0 +1,45 @@
+# Sample Luigi application written in OpenUI5
+
+## Overview
+
+This is the OpenUI5-based sample application which runs with Luigi framework.
+
+
+## Development
+
+
+To run this application, follow these steps:
+
+1. Make sure you have [UI5 CLI](https://sap.github.io/ui5-tooling/pages/CLI/) installed by executing the command: 
+    ```bash
+    npm install --global @ui5/cli
+    ```
+
+2. Install Dependencies
+    ```bash
+    npm install
```

Comment:
I've got an error here due to missing of `libs` folder during running this command `postinstall`.